kale/embed/factorization.py

```diff
@@ -301,7 +301,9 @@ def __init__(
             self.kernel_params = kernel_params
         self._label_binarizer = LabelBinarizer(pos_label=1, neg_label=-1)
         self._centerer = KernelCenterer()
+        self.fit_label = fit_label
         self.x_fit = None
+        self.enc = OneHotEncoder(handle_unknown="ignore")
```

Comment:
In this case, I would prefer to put this one inside the `fit` method rather than `__init__`. I would also recommend adding the underscore postfix (e.g., `enc_`) as it is like the default way to define additional attributes defined during fit in sklearn.